packages/mobile/src/analytics/ValoraAnalytics.test.ts

```diff
@@ -21,6 +21,9 @@ jest.mock('react-native-permissions', () => ({}))
 jest.mock('@sentry/react-native', () => ({ init: jest.fn() }))
 jest.mock('src/redux/store', () => ({ store: { getState: jest.fn() } }))
 
+const mockDeviceId = 'abc-def-123' // mocked in __mocks__/react-native-device-info.ts (but importing from that file causes weird errors)
```

Comment:
this is at least not worse than what's already happening in this test file, where we hardcode expected bundle id's, for instance, to `'org.celo.mobile.debug'`, which mirrors another mocked value from `__mocks__/react-native-device-info.ts`

Reply:
but if I'm missing some nicer way to do this I'd still be curious to hear about it

Reply:
Good, that's the way to do it. 👍 
We don't want to import from the `__mocks__` folder directly.